bims/tasks/taxa_upload.py

```diff
@@ -7,7 +7,8 @@
     TAXON,
     SCIENTIFIC_NAME,
     ORIGIN,
-    ENDEMISM
+    ENDEMISM,
+    VARIETY
```

Comment:
@Fanevanjanahary please don't add the variable here, VARIETY is optional header.